.gitignore

```diff
@@ -19,6 +19,7 @@
 /_test
```

Comment:
It's unclear to me why the Sytest job is failing and what it is complaining about, https://buildkite.com/matrix-dot-org/dendrite/builds/4671#46eca488-3a58-4493-8b43-9866ec5e9760

Reply:
The key change test is flakey annoyingly.